Network/Socket.hsc

```diff
@@ -242,1456 +227,20 @@ module Network.Socket
     , recvFrom
     , inet_addr
     , inet_ntoa
+    , htonl
+    , ntohl
+    , fdSocket
+    , socketToHandle
```

Comment:
Should these receive proper deprecation pragmas?

Reply:
I will do it.